_sass/calendar.scss

```diff
@@ -21,4 +21,7 @@
       background-color: var(--brand-color-primary) !important;
     }
   }
+  .fc-prev-button,.fc-next-button,.fc-today-button{
+    margin: 5px;
+  }
```

Comment:
This should only be aplied for screen width <= `574px`, please use media query for the same.

Also, only a bottom margin is required. Update the CSS accordingly.